code/modules/paperwork/paper_cutter.dm

```diff
@@ -23,12 +23,10 @@
 	update_appearance()
 
 /obj/item/papercutter/Destroy(force)
-	if(stored_paper)
-		stored_paper.forceMove(get_turf(src))
-		stored_paper = null
-	if(stored_blade)
-		stored_blade.forceMove(get_turf(src))
-		stored_blade = null
```

Comment:
make moving it out a part of deconstruct
and make Exited or whatever its called null the variables when they leave the papercutter